packages/patrol/lib/src/native/native_automator2.dart

```diff
@@ -240,6 +240,14 @@ class NativeAutomator2 {
     );
   }
 
+  /// Opens the URL specified by [url].
+  Future<void> openUrl({required String url}) async {
```

Comment:
Imo `url` could be a positional parameter here. It's quite clear that this method takes a url